sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/InvalidCastToInterface.cs

```diff
@@ -33,8 +34,7 @@
 namespace SonarAnalyzer.Rules.CSharp
 {
     [DiagnosticAnalyzer(LanguageNames.CSharp)]
-    [Rule(DiagnosticId)]

```

Comment:
Rule attribute is removed because we have unit tests that ensure each rule is registered in exactly one diagnostic analyzer. InvalidCast is reported from two analyzers.